src/plugins/dashboard/common/content_management/v1/types.ts

```diff
@@ -59,9 +61,14 @@ export interface SavedDashboardPanel {
   version?: string;
 }
 
+type ControlGroupAttributesV1 = Pick<
+  RawControlGroupAttributes,
+  'panelsJSON' | 'chainingSystem' | 'controlStyle' | 'ignoreParentSettingsJSON'
```

Comment:
Sometime in the future we will likely get rid of the JSON bags where we will remove them from `RawControlGroupAttributes`. But I think typescript will catch the mismatch and we can then add those types (`panelsJSON` and `ignoreParentSettingsJSON`) directly to the `ControlGroupAttributesV1` type. 

I think nothing to worry about in this PR though.